pkg/logproto/logproto.proto

```diff
@@ -162,10 +162,26 @@ message Sample {
   uint64 hash = 3 [(gogoproto.jsontag) = "hash"];
 }
 
+message SampleHistogram {
+  double  count = 1      [(gogoproto.jsontag) = "count"];
+  double  sum = 2 [(gogoproto.jsontag) = "sum"];
+	repeated HistogramBucket buckets = 3 [
+    (gogoproto.nullable) = false,
+    (gogoproto.jsontag) = "buckets"
+  ];
+}
+
+message HistogramBucket {
+	double lower = 1 [(gogoproto.jsontag) = "lower"];
+	double upper = 2 [(gogoproto.jsontag) = "upper"];
+	double count = 3 [(gogoproto.jsontag) = "count"];
+}
+
 // LegacySample exists for backwards compatibility reasons and is deprecated. Do not use.
 message LegacySample {
   double value = 1;
   int64 timestamp_ms = 2;
+  SampleHistogram histogram = 3 [(gogoproto.jsontag) = "histogram"];
```

Comment:
Why are we adding a histogram to every sample?

Reply:
To match `model.Sample` https://github.com/prometheus/common/blob/main/model/value.go#L36